industry_benchmarks/utils/tests/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
```

Comment:
Does this rely on zsh (I know it has some differences to bash)? zsh might not be on a cluster, I am 99% sure this works on bash so maybe 
```suggestion
#!/user/bin/env bash
```

Reply:
ah no! this is a temprorary file!